vortex-flatbuffers/flatbuffers/vortex-serde/footer.fbs

```diff
@@ -7,6 +7,7 @@ table Layout {
     encoding: uint16;
     buffers: [Buffer];
     children: [Layout];
+    length: uint64;
```

Comment:
This length being everywhere is very duplicative but not sure of easy solution, could be optionally in metadata